ios/brave-ios/Sources/Brave/WebFilters/FilterListStorage.swift

```diff
@@ -54,6 +54,27 @@ import Preferences
     }
   }
 
+  func updateFilterLists() async -> Bool {
```

Comment:
this will run on main, is that ok?

Reply:
I believe this should be fine. The method does this:
```
DCHECK_CALLED_ON_VALID_SEQUENCE(sequence_checker_);
```

Reply:
Okay then shit will require it always run on main then unless you use a task runner on the core side, if the `@MainActor` is ever removed from the parent type this will need to make sure its marked as MainActor again